### PR TITLE
feat(loyalties): update book schemas

### PR DESCRIPTION
**Pull request by qtotuan**

### CODE REVIEW

File: lib/v0/loyalty_systems/base.js [hunk 1]
```diff
@@ -172,5 +172,75 @@ module.exports = {
...
+        },
+        to: {
+          type: 'string',
+          description: 'Name of the unit which you are converting the amount to. This can be a 3 letter ISO currency code or any other string that corresponds to a unit from loyalty_systems.value_configurations.'
+        },
+        rate: {
+          type: 'number',
+          description: 'The exchange rate',
+          minimum: 0,
+          multipleOf: 0.01,
```
Comment (Endebert) [inline lib/v0/loyalty_systems/base.js]: Should exchange rate maybe be larger?

E.g. these sites have 5-digit decimal precision:
https://www.x-rates.com/
https://transferwise.com/gb/currency-converter/

But maybe this is a different context

Comment (qtotuan) [reply lib/v0/loyalty_systems/base.js]: Ah you discovered an interesting point :) In our case they are not "real" money exchange rates. It is for example to convert from "miles" to "EUR" or from "coffee_stamps" to "points", etc. It is completely arbitrary and maybe more like a general factor. But I know what you mean, I've thought about it, but as this is a first tentative idea, and not implemented anywhere in the th-api yet, I'd leave this for now. Later I could rename it to "conversion_factor" so no one will confuse it with actual exchange_rates.

Comment (Endebert) [reply lib/v0/loyalty_systems/base.js]: Oh I see, ok then 👌

**Pull request merged**